examples/apps/presence-tracker/src/FocusTracker.ts

```diff
@@ -106,7 +106,7 @@ export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
 			member.connections.forEach((connection) => {
 				const focus = this.getFocusPresenceForUser(userId, connection.id);
 				if (focus !== undefined) {
-					statuses.set((member as any).userName, focus);
+					statuses.set((member as any).name, focus);
```

Comment:
More proof that `as` is letting us down. How can we avoid `as`? If we can't avoid `as` can we at least avoid `any`?

Reply:
removed `as any` completely